extjsdk/src/test/java/io/vantiq/extjsdk/TestExtensionWebSocketListener.java

```diff
@@ -258,12 +259,12 @@ public void testDefaultQuery() {
         
         connectToSource(srcName, null);
         
-        ResponseBody body = createQueryMessage(new LinkedHashMap(), srcName);
+        ResponseBody body = TestListener.createQueryMessage(new LinkedHashMap(), srcName);
         listener.onMessage(body);
         Response resp = null;
         try {
-            resp = mapper.readValue(client.getLastMessage(), Response.class);
-        } catch(Exception e) { e.printStackTrace();}
+            resp = client.getLastMessageAsResponse();
+        } catch(Exception e) { e.printStackTrace(); assert false;}
```

Comment:
"assert false" is probably better done as "fail" with a message telling about the unexpected exception.  It make the code more self-documenting.